programs/programs/nc/nc_lieap/calculator.py

```diff
@@ -38,18 +38,20 @@ def household_value(self):
         # Calculate income and limits
         gross_income = self._calculate_gross_income()
         income_limit = self._calculate_income_limit()
```

Comment:
It looks like this PR changes the logic so we no longer use this custom income limit from `self._calculate_income_limit()` for calculating value, only for calculating eligibility. Just want to make sure, it's intentional to use different income limits in each place? If that's correct, can you remove this unused line?